.env.development

```
@@ -7,5 +7,8 @@ LOCAL_ETHER_BALANCE="1000000"
 CONSTRUCTOR_TOKEN_NAME="Potato"
 CONSTRUCTOR_TOKEN_SYMBOL="POT"
 CONSTRUCTOR_TOKEN_DECIMALS=0
+CONSTRUCTOR_NFT_NAME="Broccoli"
+CONSTRUCTOR_NFT_SYMBOL="BRC"
+CONSTRUCTOR_TOKEN_MAX=100
```

I believe this should be `CONSTRUCTOR_NFT_MAX` as used in the deployment script

```suggestion
CONSTRUCTOR_NFT_MAX=100
```